### PR TITLE
Adding support to Franzininho WiFi Boards - CircuitPyhton

### PR DESCRIPTION
**Pull request by FBSeletronica**
Hello,

I added VID/PIDs for Franzininho Wifi boards:
[Franzininho WiFi Wrover](https://circuitpython.org/board/franzininho_wifi_wrover/)
[Franzininho WiFi Wroom](https://circuitpython.org/board/franzininho_wifi_wroom/)

Thanks!

### CODE REVIEW

File: mu/modes/circuitpython.py [hunk 1]
```diff
@@ -72,6 +72,8 @@ class CircuitPythonMode(MicroPythonMode):
         (0x239A, 0x80AC, None, "Unexpected Maker FeatherS2"),
         (0x303A, 0x8002, None, "Unexpected Maker TinyS2"),
         (0x054C, 0x0BC2, None, "Spresense"),
+        (0x303A, 0x80AA, None, "Franzininho WiFi Wroom"),
+        (0x303A, 0x80AD, None, "Franzininho WiFi Wrover"),
```
Comment (carlosperate) [inline mu/modes/circuitpython.py:76]: Thanks! I can confirm the PIDs from the Espressif repo:
- https://github.com/espressif/usb-pids/blob/d40fe6f7369261db0f5394c5c4bf24311bc6184d/allocated-pids.txt#L178
- https://github.com/espressif/usb-pids/blob/d40fe6f7369261db0f5394c5c4bf24311bc6184d/allocated-pids.txt#L181

Comment (carlosperate) [inline mu/modes/circuitpython.py:76]: I think the reason the CI is complaining is that there are extra white spaces at the end of this line.
You can run the tests locally to ensure everything passes with: `make check` (or `python make.py check`)

Comment (FBSeletronica) [reply mu/modes/circuitpython.py]: @carlosperate I removed the extra white space.

Thanks for the feedback

**Comment by ntoll** [comment]
@FBSeletronica thank you so much for your contribution.

Just a quick note. Adafruit recently contributed a change to read board information using their `adafruit-board-toolkit` module (see #1371). As a result, the changes you propose are superseded by this new method of board detection..!

The next beta release (out tomorrow - Monday) will include these changes. Please test and feel free to give feedback if you encounter problems.

Thank you so much..! :+1:

**Comment by FBSeletronica** [comment]
@ntoll thank you. I will test.

**Pull request closed without merging**